extra/man/alacritty.5.scd

```diff
@@ -11,17 +11,15 @@ can be found at _https://toml.io/en/v1.0.0_.
 
 # LOCATION
 
-Alacritty doesn't create the config file for you, but it looks for one in the
-following locations on UNIX systems:
+Alacritty doesn't create the config file for you, but it looks for one at the
+following paths:
```

Comment:
Why was this changed?

Reply:
to have the same wording as in `alacritty(1)`

Reply:
Didn't we change this intentionally to explicitly call out unix? So shouldn't `alacritty(1)` be changed?

Reply:
yeah, one of them should be changed, so I'll change whatever you prefer.

Reply:
I prefer changing it to explicitly point these out as unix paths. So I'd change `alacritty(1)`.